### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by fdiblen**
Adds a pull request template

### CODE REVIEW

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Content looks good to me.

At https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository lowercase is used for filename. 

Uppercase probably also works, but I think it is better to follow the docs. Can you rename the file to `.github/pull_request_template.md`?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## List of related issues or pull requests
+
+Refs: #ISSUE_NUMBER
+
+
+## Describe the changes made in this pull request
+
+
+## Instructions to review the pull request
```
Comment (jspaaks) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: I prefer to not shout big captions at whoever makes the pull request, just boldface is enough

```suggestion
**List of related issues or pull requests**

Refs: #ISSUE_NUMBER

**Describe the changes made in this pull request**

**Instructions to review the pull request**
```

**Review by jspaaks — APPROVED** [review]
:trophy: :8ball:

**Review by sverhoeven — APPROVED** [review]
Nice

**Pull request merged**